modules/ossm-validate-encryption-kiali.adoc

```diff
@@ -0,0 +1,30 @@
+////
+This module included in the following assemblies:
+* service_mesh/v2x/prepare-to-deploy-applications-ossm.adoc
+////
+:_content-type: CONCEPT
+[id="ossm-validating-sidecar_{context}"]
+= Validating encryption with Kiali
+
+The Kiali console offers several ways to validate whether or not your applications, services, and workloads have mTLS encryption enabled.
+
+.Masthead icon mesh-wide mTLS enabled
+image::ossm-kiali-masthead-mtls-enabled.png[mTLS enabled]
+
+At the right side of the Masthead, Kiali shows a lock icon when the mesh has strictly enabled mTLS for the whole service mesh. It means that all communications in the mesh use mTLS.
+
+.Masthead icon mesh-wide mTLS partially enabled
+image::ossm-kiali-masthead-mtls-partial.png[mTLS partially enabled]
+
+Kiali displays a hollow lock icon when either the mesh is configured in `PERMISSIVE` mode or there is a error in the mesh-wide mTLS configuration.
+
+.Security badge
+image::ossm-kiali-graph-badge-security.png[Security badge]
+
+The *Graph* page has the option to display a *Security* badge on the graph edges to indicate that mTLS is enabled.  To enable security badges on the graph, from the *Display* menu, under *Show Badges*, select the *Security* checkbox.  When an edge shows a lock icon it means at least one request with mTLS enabled is present.  In case there are both mTLS and non-mTLS requests, the side-panel will show the percentage of requests using mTLS.
+
+The *Applications Detail Overview* page displays a *Security* icon on the graph edges that have mTLS enabled.
```

Comment:
If this is referring to the little mini-graph that shows up, the graph icons are the same as that on the Graph page. So perhaps you should mention something to that effect (because this means "When an edge shows a lock icon it means at least one request with mTLS enabled is present, but there could be both mTLS and non-mTLS requests").